interface/forms/soap/templates/general_new.html

```diff
@@ -4,49 +4,79 @@
 <title>{xl t='SOAP'|escape:'html'}</title>
 
 {headerTemplate}
+{literal}
+ <style type="text/css" title="mystyles" media="all">
+ <!--
```

Comment:
@zbig01 and @robertdown ,
1. If we migrate legend and fieldset styling to the external stylesheets(this would be easy to do and would then allow per style sheet customizations of these elements), then would we be able to remove the internal stylesheets?
2. And regarding the media queries in styling, why is this needed. Doesn't bootstrap manage that by itself?